opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -462,5 +462,31 @@ namespace Opm
                 }
             }
         }
+
+        for (int w = 0, nw = wells->number_of_wells; w < nw; ++w) {
+            WellControls* ctrl = wells->ctrls[w];
+            const bool is_producer = wells->type[w] == PRODUCER;
+            if (!is_producer && wells->name[w] != 0) {
+                WellMap::const_iterator i = wmap.find(wells->name[w]);
+                if (i != wmap.end()) {
+                    WellConstPtr wp = i->second;
+                    const WellInjectionProperties& i = wp->getInjectionProperties(step);
```

Comment:
May I suggest a different variable name here?  This `i` shadows the `WellMap::const_iterator` defined three lines up.